wfe2/stats.go

```diff
@@ -43,9 +46,18 @@ func initStats(scope metrics.Scope) wfe2Stats {
 	)
 	scope.MustRegister(csrSignatureAlgs)
 
+	improperECFieldLengths := prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "improperECFieldLengths",
+			Help: "Number of EC keys with improper X and Y lengths",
```

Comment:
nit: I think this should specify that its the number of _account_ keys.
```suggestion
			Help: "Number of account EC keys with improper X and Y lengths",
```

Reply:
Resolved in 8a718ef